lib/srv/usermgmt.go

```diff
@@ -64,10 +64,12 @@ type HostUsersBackend interface {
 	Lookup(name string) (*user.User, error)
 	// LookupGroup retrieves a group by name.
 	LookupGroup(group string) (*user.Group, error)
+	// LookupGroup retrieves a group by name.
+	LookupGroupId(group string) (*user.Group, error)
```

Comment:
```suggestion
	// LookupGroupID retrieves a group by name.
	LookupGroupID(group string) (*user.Group, error)
```

initialisms should be all caps